src/server/data/automatedTestingData.js

```diff
@@ -41,11 +60,12 @@ async function generateSineTestingData(frequency = 15, amplitude = 1) {
 		timeStep: { minute: frequency },
 		periodLength: DEFAULT_OPTIONS.periodLength,
 		maxAmplitude: amplitude,
-		filename: path.join(__dirname, '../test/db/data/automatedTests/' + frequency.toString()
-			+ 'Freq' + amplitude.toString() + 'AmpSineTestData.csv'),
 		normalize: true
 	};
-	await generateSine(startDate, endDate, options);
+	// Store data in a variable
+	const generatedData = await generateSine(startDate, endDate, options);
+	// Return the generatedData in the function
+	return generatedData;
 }
 
 /**
```

Comment:
Update this comment so say it puts the values into the database rather than saves to file.